src/app/modules/spcp/spcp.util.ts

```diff
@@ -114,6 +115,42 @@ export const extractJwt = (
   }
 }
 
+export const createSingpassParsedResponses = (
+  uinFin: string,
+): ProcessedSingleAnswerResponse[] => {
+  return [
+    {
+      _id: '',
+      question: 'SingPass Validated NRIC',
+      fieldType: 'authenticationSp' as BasicField,
```

Comment:
Should we add typings for this? Then we can update fieldType to be a union of these typings

Reply:
idk, I wasn't really sure how to deal with this. one option is to create types for 'authenticationSp' and 'authenticationCp'. but actually this fieldType isn't even used anywhere, I just left it in because `fieldType` is a required field for `ProcessedFieldResponse`. I was hoping that when we eventually disentangle `email-submissions.server.controller.js`, we'll find a better way to handle SPCP fields than creating fake fieldTypes.

Reply:
tl;dr I left it like this because I thought types wouldn't be useful